docs/server-security.md

```diff
@@ -0,0 +1,196 @@
+---
+id: server-security
+title: Temporal Server security
+sidebar_label: Security
+---
+
+## Overview
+
+A secured Temporal server has its network communication encrypted and has authentication and authorization protocols set up for API calls made to it. Without these your server could be accessed by unwanted entities.
```

Comment:
[suggestion] "What is documented on this page are the inbuilt opt-in security measures in Temporal, however users may also choose to design their own security architecture with reverse proxies or running unsecured inside of a VPC environment."

Reply:
Dig it - appended with this.